### ISSUE TITLE
Build issue with TypeScript project "botbuilder does not export member 'Promiseable'"

### ISSUE THREAD

**Issue by liliankasem** [great-lakes/botbuilder-wolf#165]
**Describe the bug**
I get a build error after adding wolf to my existing bot project. 

`node_modules/wolf-core/dist/middlewares/wolfMiddleware.d.ts:1:42`

It is complaining that the botbuilder node module does not export a member called `Promiseable`

**To Reproduce**
1. Follow README instructions to add wolf to an existing (typescript) bot v4 project
2. Run `tsc`

**Screenshots**
<img width="1311" alt="screen shot 2019-03-04 at 6 21 33 pm" src="https://user-images.githubusercontent.com/2198905/53776269-6a485b00-3eaa-11e9-926f-d459cb584f0c.png">

<img width="1318" alt="screen shot 2019-03-04 at 6 09 38 pm" src="https://user-images.githubusercontent.com/2198905/53776004-71229e00-3ea9-11e9-803e-6b07e46184ec.png">

Bonus -> can't write abilities.d.ts file:
<img width="1150" alt="screen shot 2019-03-04 at 6 22 17 pm" src="https://user-images.githubusercontent.com/2198905/53776295-7a603a80-3eaa-11e9-8420-d9adc31b53d5.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.13.16
 - Browser: Chrome
 - Wolf-core version: 2.5.2

**Comment by kevinleung23** [comment]
Hey @liliankasem thanks for filing this. Sorry this didn't work out of the box for you, it looks like `botbuilder` package no longer exports the `Promisable` type from the version we were referencing. We will look into getting `2.5` running..

Id recommend using our `alpha` version of [wolf-botbuilder](https://github.com/wolf-packs/wolf-botbuilder) which will give you more flexibility utilizing wolf as a function call instead of relying on using the botbuilder middleware pattern.

Since we are currently developing and improving both `wolf-core` and `wolf-botbuilder`, I'd recommend locking into `wolf-botbuilder@1.0.0-alpha.8` which is locked on `wolf-core@3.0.0-alpha.10`. This will prevent pulling any new changes we we are making daily on `@alpha`. If you look at [wolf-botbuilder/examples](https://github.com/wolf-packs/wolf-botbuilder/tree/master/examples) there should be working examples..

Hope this helps, please let me know if you run into any issues/have suggestions.

**Comment by liliankasem** [comment]
Thanks Kevin! I'll look into wolf-botbuilder and play around with that.

**Comment by kevinleung23** [comment]
We'd welcome any feedback, glad you're checking out Wolf! Hoping to get v3 stable and out soon. :)

**Comment by liliankasem** [comment]
Yeah of course, happy to contribute where I can. Currently just trying to evaluate if this will be a good fit for an upcoming project :)
